pyproject.toml

```diff
@@ -106,3 +106,4 @@ reportUnboundVariable = "warning"
 [tool.pytest.ini_options]
 filterwarnings = ['ignore:.*The environment .* is out of date.*']
 # filterwarnings = ['ignore:.*step API.*:DeprecationWarning']
+addopts = --ignore=docs/tutorials/using_datasets/behavioral_cloning.py
```

Comment:
This was the right thing to do just the wrong syntax, addopts has to be a list with quotes around each option, I'll push a commit fixing it

Reply:
Oh thank you; I am investigating locally why it doesn't end in a reasonable amount of time, so maybe we can keep the test

Reply:
I have a pr in the pz repo testing an alternative more manual method of testing docs but there’s some issues with it, it may be more scalable/faster tho I can bump here if I get it working